_ko/tour/tuples.md

```diff
@@ -5,13 +5,79 @@ title: Tuples
 discourse: true
 
 partof: scala-tour
-num: 
+
+num: 6
 language: ko
 
 next-page: mixin-class-composition
 previous-page: traits
+topics: tuples
 
+redirect_from: "/tutorials/tour/tuples.html"
 ---
 
-(this section of the tour has not been translated yet. pull request
-with translation welcome!)
\ No newline at end of file
+스칼라에서 하나의 튜플은 정해진 갯수의 각각 구별된 타입을 갖는 엘리먼트들을 포함하는 하나의 값이다. 튜플은 불변적이다.
+
+튜플은 특히 메소드로부터 여러개의 값을 리턴하는데 편리하게 사용할 수 있다.
+
+두개의 엘리먼트를 갖는 튜플은 다음과 같이 생성할 수 있다:
+
+```tut
+val ingredient = ("Sugar" , 25)
+```
+
+위 코드는 `String`과 `Int` 엘리먼트를 포함하는 튜플을 생성한다.
+
+`ingredient`의 추론된 타입은 `Tuple2[String, Int]`의 약칭인 `(String, Int)`이다.
+
+튜플들을 나타내기 위해서 Scala에서는 다음과 같은 형태의 클래스들을 사용한다: `Tuple2`, `Tuple3`, ... ,`Tuple22`.
+각 클래스는 파라미터 타입의 갯수 만큼 엘리먼트들을 가지고 있다.
+
+## 엘리먼트 접근하기
+
+튜플의 엘리먼트에 접근하기 위한 한가지 방법은 위치로 접근하는 것이다. 각 개별적인 엘리먼트들은 `_1`, `_2`, ... 이와 같은 방법으로 이름지어진다.
```

Comment:
When translating into Korean, it seems natural to use active type rather than passive type.

각 개별적인 엘리먼트들은 `_1`, `_2`, ... 이와 같은 방법으로 이름지어진다.

How about this?
각 개별적인 엘리먼트들은 `_1`, `_2`, ... 와 같은 이름을 갖는다.

Reply:
I think 각 and 개별 is duplicate semantically.

각 개별적인 엘리먼트들은

How about this?

각 요소들은

Reply:
@horace-velmont @byeonggukgong  Both are good suggestions!